src/server/configs/socket.config.ts

```diff
@@ -0,0 +1,31 @@
+import dotenv from 'dotenv';
+import { Server } from "socket.io";
+import http from 'http';
+import { app } from './root.config.js'
+
+dotenv.config();
+
+const server = http.createServer(app);
+const io = new Server(server, {
+  cors: { origin: ["http://localhost:4200", "http://localhost:8080"] },
+});
+
+const access_secret = process.env.ACCESS_SECRET as string;
+const port = process.env.PORT || 3000;
+
+export const socketConfig = {
+    secret: access_secret,
```

Comment:
this access_secret also doesnt belong here.. the access_secret has nothing to do with sockets but to do with signing and verifying a json web token.. this maybe could be part of the confusion with "socket authentication"

Reply:
resolved